Dreary/README.md

```diff
@@ -17,15 +17,23 @@
 ### Golden
 ![Golden Screenshot](golden.png)
 
+### Graytone-Blue
```

Comment:
I'm not sure I want to merge this color scheme, it looks quite similar to Deeper

Reply:
@morpheusthewhite Yeah, I mean my goal with this is that the color scheme works with any text and subtext color, so eventually i can just make a ton of schemes or even make an extension to let the user pick their own colors with ease.

Reply:
Yeah in my opinion the extension is much better. You will never be able to cover all the color schemes preferences otherwise

Reply:
@morpheusthewhite Removed graytone-red, but keeping graytone blue as I think it looks rather nice

Reply:
Sorry, I don't want to look rude or stubborn but I think it's not worth adding that theme since it is too similar to Deeper

Reply:
@morpheusthewhite It's all good, just an opinion. To be honest, although it does have a resemblance to deeper because of the text color, i think the milder gray makes a fairly big difference in how the theme itself is presented. Since I removed the deep scheme entirely, I think graytone-blue fills that role much better and matches rather well with the bib color scheme and golden color scheme.